### PR TITLE
64-bit Compile

### PR DESCRIPTION
**Pull request by martinfinke**
With these changes, I could compile Max in 64-bit. I used Xcode 11.2.1 on macOS 10.14.6.

I've committed the built Frameworks as well, because that was the most work. It's a separate commit though (12dbdd9842caadb861a1791ba2fcf3a1891cb839) so you can discard that one if you don't want them. Of course it would be best to not rely on pre-built versions, so I also have a PR for the changes in AudioFrameworks: sbooth/AudioFrameworks/pull/11

I'm sorry about the premature quality of these PRs. I unfortunately don't have enough time to make it more clean. I mainly just needed a 64-bit version, and thought these changes could be a good starting point.

### CODE REVIEW

File: Max.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -506,14 +506,14 @@
 		32C8F0ED10632AB0004AB74F /* GaplessUtilities.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = GaplessUtilities.m; sourceTree = "<group>"; };
 		32CA4F630368D1EE00C91783 /* Max_Prefix.pch */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = Max_Prefix.pch; sourceTree = "<group>"; };
 		8C0406480B5C582E00648158 /* nb_NO */ = {isa = PBXFileReference; lastKnownFileType = text.rtf; name = nb_NO; path = nb_NO.lproj/Credits.rtf; sourceTree = "<group>"; };
-		8C0406490B5C584000648158 /* nb_NO */ = {isa = PBXFileReference; fileEncoding = 10; lastKnownFileType = text.plist.strings; lineEnding = 0; name = nb_NO; path = nb_NO.lproj/CompactDisc.strings; sourceTree = "<group>"; };
+		8C0406490B5C584000648158 /* nb_NO */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.strings; lineEnding = 0; name = nb_NO; path = nb_NO.lproj/CompactDisc.strings; sourceTree = "<group>"; };
```
Comment (martinfinke) [inline Max.xcodeproj/project.pbxproj]: There was a warning in Xcode, I hope this change is ok (didn't test and don't speak Norwegian unfortunately).

File: Metadata/AudioMetadata.mm [hunk 2]
```diff
@@ -47,6 +47,24 @@
 
 #include <wavpack/wavpack.h>
 
+using namespace APE;
+
+static TagLib::ID3v2::FrameListMap frameListMap(const TagLib::Tag* tag)
```
Comment (martinfinke) [inline Metadata/AudioMetadata.mm]: This and `getFrameList` should probably be moved somewhere else, and not duplicated.

**Comment by sbooth** [comment]
Thanks for this. Max now builds correctly for 64-bit.

**Comment by martinfinke** [comment]
Glad I could help 😊 and thank you for your work.

**Pull request closed without merging**